backend/api/swagger.yml

```diff
@@ -513,6 +513,145 @@ components:
     Property:
       type: object
       properties:
-        id:
+        landlord_id:
+          type: string
+          format: uid
+          description: ID of landlord
+          example: 359627C2-C1E2-4CC3-89BD-3D33E9EC2AAB
```

Comment:
@thinkharderdev @ericssy 

property can be created by a landlord or a navigator. We should add a few fields here. `created_by [user id]`, `last_updated_by [user_id]` 

It would also be prudent to prefix the type of id to each random string (e.g. `landlord-359627C2-C1E2-4CC3-89BD-3D33E9EC2AAB`)